types/aws-lambda/trigger/kinesis-firehose-transformation.d.ts

```diff
@@ -10,7 +10,8 @@ export type FirehoseTransformationCallback = Callback<FirehoseTransformationResu
 // Examples in the lambda blueprints
 export interface FirehoseTransformationEvent {
     invocationId: string;
-    deliveryStreamArn: string;
+    deliveryStreamArn?: string | undefined;
```

Comment:
Technically a breaking change: will AWS actually send an undefined / missing value for this property? If so, this is good!

Reply:
I have double checked and reverted the change here. It looks like it is always present.